### DISCUSSION TITLE
Clarify pronounciation

### DISCUSSION THREAD

**Discussion by skorokithakis** [tokio-rs/tokio#3818] [Q&A] [answered]
I think it's crucial to clarify whether we should pronounce the name of the library as "Tokyo" or as "tok I/O". Someone please advise!

**Comment by memoryruins** [comment, marked as answer]
It's like "tokyo"; a similar discussion is in https://github.com/tokio-rs/tokio/discussions/3668

**Reply by skorokithakis** [reply]
Ah, thanks, I searched both issues and here but didn't find that one. Thanks!

**Reply by memoryruins** [reply]
https://youtu.be/MZyleK8elPk is a recent presentation about tokio. it's pronounced near the very beginning.

**Comment by carllerche** [comment]
Tokio is an alternate spelling of Tokyo (the city) and the Tokio logo is inspired by the Metropolitan crest: https://en.wikipedia.org/wiki/Symbols_of_Tokyo

**Reply by skorokithakis** [reply]
Makes sense, thanks! Adding this in the docs or somewhere would be very helpful, I think.
